[ASM] - EXPANDR-6449 - Fixing GCP Bug (#30126)
demisto/content: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_3.md

#### Playbooks

##### Cortex ASM - Prisma Cloud Enrichment

> - Fixed an issue with no network tags logic. Now we have an additional check for this.

Reviewer: ```suggestion
Fixed an issue with no network tags logic. Now there is an additional check for this.
```
